src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -178,21 +182,21 @@ final class ScalaCheckFramework extends Framework {
 
   val name = "ScalaCheck"
 
-  def fingerprints: Array[Fingerprint] = Array(
+  def fingerprints: Array[Fingerprint | Null] = Array(
```

Comment:
A Scala method overrides some Java method. We need to add |Null to the method's parameter types to make the override still work.